### PR TITLE
convert nightly into new config

### PR DESCRIPTION
**Pull request by tomsilver**
results:
```
Git commit hashes seen in results/:
fb7506fa7cdc7619684efc0f35dc94d75605c547


AGGREGATED DATA OVER SEEDS:
                                  NUM_TRAIN_TASKS    NUM_SOLVED AVG_NUM_PREDS  AVG_TEST_TIME  AVG_NODES_CREATED       LEARNING_TIME  NUM_SEEDS
EXPERIMENT_ID                                                                                                                                 
coffee-option_learning             1000.00 (0.00)  49.30 (1.00)  13.00 (0.00)  46.86 (10.41)       41.70 (4.63)      1001.22 (8.43)         10
cover-invent_allexclude              50.00 (0.00)  49.30 (0.90)   5.90 (0.54)    0.01 (0.00)        5.00 (0.23)      153.91 (15.31)         10
cover-nsrt_learning                  50.00 (0.00)  49.50 (0.50)   5.00 (0.00)    0.06 (0.02)        6.66 (0.36)        20.71 (1.65)         10
cover-option_learning              1000.00 (0.00)  49.70 (0.64)   5.00 (0.00)    1.95 (2.73)        7.15 (0.42)       374.63 (3.77)         10
cover-oracle                         50.00 (0.00)  50.00 (0.00)   5.00 (0.00)    0.04 (0.01)        6.66 (0.32)         0.00 (0.00)         10
doors-option_learning              1000.00 (0.00)  49.40 (0.80)   7.00 (0.00)    1.31 (0.38)       40.78 (4.75)    1287.88 (404.92)         10
painting-invent_allexclude           50.00 (0.00)  47.30 (2.33)  21.91 (2.35)    0.18 (0.05)    298.16 (119.47)  17463.90 (4124.80)         10
painting-nsrt_learning               50.00 (0.00)  49.50 (0.50)  15.00 (0.01)    0.42 (0.11)   1723.91 (697.65)        73.50 (6.12)         10
painting-oracle                      50.00 (0.00)  50.00 (0.00)  15.00 (0.01)    0.39 (0.10)   2301.79 (738.99)         0.00 (0.00)         10
playroom-nsrt_learning               50.00 (0.00)  50.00 (0.00)  19.00 (0.00)    0.16 (0.00)       44.28 (1.11)       219.48 (5.14)         10
playroom-oracle                      50.00 (0.00)  50.00 (0.00)  19.00 (0.00)    0.16 (0.00)       82.54 (3.67)         0.00 (0.00)         10
pybullet_blocks-invent_allexclude    50.00 (0.00)  49.40 (0.66)   6.20 (0.75)    0.47 (0.13)  4466.73 (2295.45)    2392.69 (163.06)         10
pybullet_blocks-nsrt_learning        50.00 (0.00)  49.60 (0.49)   5.00 (0.00)    0.42 (0.12)  4201.33 (2124.80)        20.51 (1.64)         10
pybullet_blocks-oracle               50.00 (0.00)  49.80 (0.40)   5.00 (0.00)    0.41 (0.13)  4259.69 (2224.80)         0.00 (0.00)         10
pybullet_cover-invent_allexclude     50.00 (0.00)  47.70 (1.95)   6.00 (0.45)    0.05 (0.01)        4.94 (0.22)      258.58 (21.53)         10
pybullet_cover-nsrt_learning         50.00 (0.00)  47.80 (1.99)   5.00 (0.00)    0.10 (0.02)        6.64 (0.33)        22.44 (1.58)         10
pybullet_cover-oracle                50.00 (0.00)  47.20 (1.40)   5.00 (0.00)    0.08 (0.01)        6.66 (0.32)         0.00 (0.00)         10
stick_button-nsrt_learning          500.00 (0.00)  47.70 (1.10)   6.00 (0.00)    0.28 (0.04)     431.54 (52.73)        18.64 (1.95)         10
stick_button-option_learning       1000.00 (0.00)  41.00 (2.49)   6.00 (0.00)  59.63 (10.33)     553.80 (86.45)      708.90 (10.82)         10
stick_button-oracle                 500.00 (0.00)  45.20 (1.99)   6.00 (0.00)    0.24 (0.03)     431.64 (58.97)         0.00 (0.00)         10
tools-invent_allexclude             200.00 (0.00)  47.60 (2.01)  29.20 (6.08)    0.42 (0.35)  1864.49 (1723.74)  20392.91 (3909.16)         10
tools-nsrt_learning                 200.00 (0.00)  49.40 (0.66)  15.00 (0.00)    0.51 (0.09)  4779.29 (1006.97)     220.12 (154.49)         10
tools-oracle                        200.00 (0.00)  50.00 (0.00)  15.00 (0.00)    0.44 (0.09)  4733.39 (1155.01)         0.00 (0.00)         10
```

### CODE REVIEW

File: scripts/configs/nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    FLAGS: {}
+ENVS:
+  playroom:
+    NAME: "playroom"
+    FLAGS: {}
+  stick_button:
+    NAME: "stick_button"
+    FLAGS:
+      timeout: 300  # requires longer timeout
+      num_train_tasks: 500  # requires more data
```
Comment (ronuchit) [inline scripts/configs/nightly.yaml:58]: say default is 50 here too?

File: scripts/configs/nightly.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      min_perc_data_for_nsrt: 1  # requires filtering
+ARGS: []
+FLAGS: {}
+START_SEED: 456
+NUM_SEEDS: 10
+...
```
Comment (ronuchit) [inline scripts/configs/nightly.yaml:64]: what does the ... do?

File: scripts/configs/nightly.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+# option learning
+---
+APPROACHES:
+  option_learning:
+    NAME: "nsrt_learning"
+    FLAGS:
+      option_learner: "direct_bc"
```
Comment (ronuchit) [inline scripts/configs/nightly.yaml:72]: what's the difference between putting these here vs in the general flags after the envs?

**Review by ronuchit — APPROVED** [review]
very nice

File: scripts/configs/nightly.yaml (hunk 2 above)
Comment (tomsilver) [reply scripts/configs/nightly.yaml:64]: it marks the end of one yaml file (seems redundant i know /shrug)

File: scripts/configs/nightly.yaml (hunk 3 above)
Comment (tomsilver) [reply scripts/configs/nightly.yaml:72]: there's no difference, but some of these flags at least feel more related to the approach. i'll move others to the general section

**Pull request merged**